src/ansys/sherlock/core/parts.py

```diff
@@ -209,7 +209,7 @@ def update_parts_list(
             "Card",
             "Sherlock Part Library",
             PartsListSearchMatchingMode.BOTH,
-            PartsListSearchDuplicationMode.ERROR,
+            PartsListSearchDuplicationMode.ERROR

```

Comment:
The comma should be re-added to be consistent with the code style.

Reply:
Done